storybookjs/storybook
# #1750 CRA Example cleanup - knobs and notes

Issue:

#1750 example app too messy

## What I did
reorganized knobs and notes
## How to test

Is this testable with jest or storyshots?
yes
Does this need a new example in the kitchen sink apps?
yes
Does this need an update to the documentation?
no
If your answer is yes to any of these, please make sure to include it in your PR.
